tools/schedule-migration/main.go

```diff
@@ -81,6 +89,38 @@ func main() {
 				}
 			}
 			new.Spec = append(new.Spec, yaml.MapItem{Key: getKeyName(old.Kind), Value: newSpec})
+		} else if findSchedule && findContainerName {
+			var cron string
+			for _, item := range schedule {
+				if item.Key == "cron" {
+					cron = item.Value.(string)
+				}
+			}
+			new.ApiVersion = old.ApiVersion
+			new.Metadata = old.Metadata
+			new.Kind = "Schedule"
+			new.Spec = append(new.Spec, yaml.MapItem{Key: "schedule", Value: cron})
+			new.Spec = append(new.Spec, yaml.MapItem{Key: "type", Value: toNewKind(old.Kind)})
+			new.Spec = append(new.Spec, yaml.MapItem{Key: "historyLimit", Value: 5})
+			new.Spec = append(new.Spec, yaml.MapItem{Key: "concurrencyPolicy", Value: "Forbid"})
+			new.Spec = append(new.Spec, yaml.MapItem{Key: "containerNames", Value: containerNames})
+			var newSpec yaml.MapSlice
+			for _, item := range old.Spec {
+				if item.Key != "scheduler" && item.Key != "containerName" {
+					newSpec = append(newSpec, item)
+				}
+			}
+			new.Spec = append(new.Spec, yaml.MapItem{Key: getKeyName(old.Kind), Value: newSpec})
```

Comment:
do not write too many duplicated codes. :cry: 

please reuse the codes in last if branch

Reply:
ok, i will modify it.